install_config/install/prerequisites.adoc

```diff
@@ -806,6 +806,53 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+Sometimes a container's log file
+(*_/var/lib/docker/containers/HASH/HASH-json.log_*) can increase to a
```

Comment:
"(the /var/lib/docker/containers/<hash>/<hash>-json.log file on the node where the container is running)